frontend/src/views/WheresMyGene/components/GeneSearchBar/components/QuickSelect/index.tsx

```diff
@@ -176,13 +176,14 @@ export default function QuickSelect<
         };
 
   const handleClose = (
-    _: React.ChangeEvent<Record<string, never>>,
+    e: React.ChangeEvent<Record<string, never>>,
     reason: AutocompleteCloseReason
   ) => {
     if (reason === "toggleInput") {
       return;
     }
-    setOpen(false);
+    // @ts-ignore: relatedTarget is a valid property as the event will sometimes be a FocusEvent depending on the action taken.
```

Comment:
I am not sure how to do this. The change handler expects a specific type but it's getting a different type (FocusEvent) depending on the action.

Reply:
I had to set it to `any` type because `e.nativeEvent` is typed as generic `Event` when we need it to be a `FocusEvent`. Casting it to `FocusEvent` doesn't work because there's no sufficient overlap in the interfaces.